lib/src/radio-group/RadioGroup.tsx

```diff
@@ -42,12 +42,21 @@ const DxcRadioGroup = React.forwardRef<RefType, RadioGroupPropsType>(
     const colorsTheme = useTheme();
     const translatedLabels = useTranslatedLabels();
 
-    const optionalItem = {
-      label: optionalItemLabel || translatedLabels.radioGroup.optionalItemLabelDefault,
-      value: "",
-      disabled,
-    };
-    const innerOptions = useMemo(() => (optional ? [...options, optionalItem] : options), [optional, options]);
+    const innerOptions = useMemo(
+      () =>
+        optional
+          ? [
+              ...options,
+              {
+                label: optionalItemLabel ?? translatedLabels.radioGroup.optionalItemLabelDefault,
```

Comment:
Changing || to ?? will change the accepted values for optionalItemLabel.
For example the optionalItemLabel="" will be accepted instead of defaulting to N/A.

Reply:
I thought about that. I see no impediment to why we cannot accept it as a valid value (the empty string value). Only `undefined` would add the default value to the prop.